source/App.cpp

```diff
@@ -87,34 +86,42 @@ void App::onInit() {
 	updateMouseSensitivity();			// Update (apply) mouse sensitivity
 	updateSessionDropDown();			// Update the session drop down to remove already completed sessions
 	updateSessionPress();				// Update session to create results file/start collection
+
+	setFrameDuration(m_wallClockTargetDuration, REAL_TIME);
 }
 
 /** Handle then user settings window visibility */
 void App::openUserSettingsWindow() {
-    m_userSettingsMode = true;
-    m_userSettingsWindow->setVisible(m_userSettingsMode);
+	m_userSettingsMode = true;
+	m_userSettingsWindow->setVisible(m_userSettingsMode);
+}
+
+/** Handle then user settings window visibility */
+void App::closeUserSettingsWindow() {
+	m_userSettingsMode = false;
+	m_userSettingsWindow->setVisible(m_userSettingsMode);
 }
 
 /** Update the mouse mode/sensitivity */
 void App::updateMouseSensitivity() {
-    const UserConfig* user = userTable.getCurrentUser();
-    // Converting from mouseDPI (dots/in) and sensitivity (cm/turn) into rad/dot which explains cm->in (2.54) and turn->rad (2*PI) factors
-    // rad/dot = rad/cm * cm/dot = 2PI / (cm/turn) * 2.54 / (dots/in) = (2.54 * 2PI)/ (DPI * cm/360)
-    const double mouseSensitivity = 2.0 * pi() * 2.54 / (user->cmp360 * user->mouseDPI);
-    const shared_ptr<FirstPersonManipulator>& fpm = dynamic_pointer_cast<FirstPersonManipulator>(cameraManipulator());
-    if (m_userSettingsMode) {
-        // Set to 3rd person (i.e. show the mouse cursor)
-        fpm->setMouseMode(FirstPersonManipulator::MouseMode::MOUSE_DIRECT_RIGHT_BUTTON);
-    }
-    else {
-        // Set to first-person mode (i.e. hide the mouse cursor)
-        fpm->setMouseMode(FirstPersonManipulator::MouseMode::MOUSE_DIRECT);
-    }
+	const UserConfig* user = userTable.getCurrentUser();
+	// Converting from mouseDPI (dots/in) and sensitivity (cm/turn) into rad/dot which explains cm->in (2.54) and turn->rad (2*PI) factors
+	// rad/dot = rad/cm * cm/dot = 2PI / (cm/turn) * 2.54 / (dots/in) = (2.54 * 2PI)/ (DPI * cm/360)
+	const double pixelsToRadians = 2.0 * pi() * 2.54 / (user->cmp360 * user->mouseDPI);
```

Comment:
DPI is also not the most correct term, though it is commonly used. I believe the technically correct term is CPI (cycles per inch), but I've only seen a couple mouse manufacturers actually use that term, even though I believe many sensor companies use CPI in their spec sheets. `dotsToRadians` sounds like a function. Maybe `radiansPerDot` is a better name for the variable. Since it's related to the mouse input to camera control, maybe `cameraRadiansPerMouseDot` is the most verbose name for it.

Reply:
Yes, I much perfer the more explicit radiansPerDot - there's no ambiguity of an implied multiply.